t/146-tcp-socket-settimeouts.t

```diff
@@ -0,0 +1,516 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use Test::Nginx::Socket::Lua;
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 3 + 1);
+
+$ENV{TEST_NGINX_RESOLVER} ||= '8.8.8.8';
+
+#log_level 'warn';
+log_level 'debug';
+
+no_long_string();
+#no_diff();
+run_tests();
+
+__DATA__
+
```

Comment:
Will you please use larger timeout values to ensure these tests also pass in the valgrind testing mode? Some tests are failing in the valgrind mode on my side :) Thanks!